app/controllers/close_contacts_controller.rb

```diff
@@ -42,4 +43,15 @@ def update
                                created_by: current_user.email,
                                comment: "User edited a close contact (ID: #{cc.id}).")
   end
+
+  # Delete an existing close contact record
+  def destroy
+    redirect_to root_url && return unless current_user.can_create_patient_close_contacts?
+    cc = current_user.get_patient(params.permit(:patient_id)[:patient_id])&.close_contacts&.find_by(id: params.permit(:id)[:id])
+    redirect_to root_url && return if cc.nil?
+    cc.destroy
+    History.close_contact_edit(patient: params.permit(:patient_id)[:patient_id],
+                               created_by: current_user.email,
+                               comment: "User deleted a close contact (ID: #{params.permit(:id)[:id]}).")
```

Comment:
This comment text needs to include all contact fields (see @mayerm94's delete labs PR) since this will be the only reference of the deleted close contact

Reply:
Be sure to include the delete_reason here as well

Reply:
I'm not sure id is necessary for the sake of the history - @holmsie should this be included?